packages/amplify_storage_s3/ios/Classes/AmplifyStorageOperations.swift

```diff
@@ -141,16 +144,55 @@ public class AmplifyStorageOperations {
         return itemAsDictionary
     }
     
-    private static func prepareError(flutterResult: @escaping FlutterResult, msg: String, localizedError: String?, recoverySuggestion: String?) {
-        var errorDetails: [String: Any] = [:]
-        var platformErrorDetails: [String: String] = [:]
-        
-        platformErrorDetails["platform"] = "iOS"
-        platformErrorDetails["localalizedError"] = localizedError
-        platformErrorDetails["recoverySuggestion"] = recoverySuggestion
-        
-        errorDetails["PLATFORM_EXCEPTIONS"] = platformErrorDetails
-        
-        flutterResult(FlutterError(code: "AmplifyException", message: msg, details: errorDetails))
+    private static func prepareError(flutterResult: @escaping FlutterResult, storageError: StorageError) {
+        let details: Dictionary<String, String> = createSerializedError(storageError: storageError);
+        logErrorContents(error: storageError)
+           ErrorUtil.postErrorToFlutterChannel(
+               result: flutterResult,
+               errorCode: "StorageException",
+               details: details)
```

Comment:
There seems to be some indentation issues here.  Default Swift indentations is to align the parameters of the function call rather than following the "continue indentation".

Also we probably don't need `;` in swift?

Reply:
+1 about removing the `;`

Reply:
Fixex